example/index.html

```diff
@@ -100,73 +181,37 @@
               gl_FragColor = vec4(sample.xyz, sample.w);
             }
         </script>
-    
-        
-        <script defer src="polyfills/harmony-collections.js"></script>
-        
         
+        <script src="polyfills/harmony-collections.js"></script>
         <script defer src="app.js"></script>
-        
-        
-        <style>
-            body{
-                display: flex;
-                align-items: center;
-                justify-content: center;
-            }
-            
-            menu{
-                position: absolute;
-                top: 0;
-                left: 0;
-                right: 0;
-                
-                display: flex;
-                align-items: center;
-                justify-content: center;
-            }
-            
-            menu button{
-                font-family: "DejaVu Serif";
-                padding: 1em 2em;
-                background-color: white;
-                margin: 0.5em;
-
-                cursor: pointer;
-
-                text-transform: uppercase;
 
-                width: 20%;
-
-                text-align: center;
-            }
-
-            menu button:disabled{
-                background-color: #AAA;
-                color: white;
-
-                text-decoration: line-through;
-
-                cursor:default;
-            }
-            
-            #view{
-                width: 60%;
-                height: 60%;
-                
-                background-color: indianred;
-            }
-            
-        </style>
-        
+        <script id="metadata" type="application/json"></script>
     </head>
     <body>
-        <menu>
-            <button id="light">Add green light</button>
-            <button id="mesh">Add red cube mesh</button>
-            <button id="controls">Switch to first person controls</button>
-        </menu>
-        
+        <div id="splash-screen">
```

Comment:
Why are the buttons gone?
Well... I guess they don't matter too much anyway at this point...